R/methods-epi_archive.R

```diff
@@ -59,9 +59,10 @@
 #'
 #' @importFrom data.table between key
 #' @export
-epix_as_of <- function(x, version, min_time_value = -Inf, all_versions = FALSE,
+epix_as_of <- function(x, version = NULL, min_time_value = -Inf, all_versions = FALSE,
                        max_version = deprecated()) {
   assert_class(x, "epi_archive")
+  version <- version %||% min(max(x$DT$time_value), x$versions_end)
```

Comment:
Isn't this going to chop off the latest version available?  `max(x$DT$time_value)` is `< x$versions_end` for most surveillance data.  Could we do just `x$versions_end`?

(Also, thoughts on introducing a new function for this rather than a default?)

Reply:
Also, due to that vctrs bug, I think if `min` stays, it needs to be something like `min(c(max(x$DT$time_value), x$versions_end))` for now.